src/Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -1,9 +0,0 @@
-namespace NServiceBus
-{
-    [System.ObsoleteAttribute(@"NServiceBus is now providing support for logging libraries through the Microsoft.Extensions.Logging abstraction. Remove the NServiceBus.NLog package. Install the NServiceBus.Extensions.Logging and NLog.Extensions.Logging packages instead. Will be treated as an error from version 4.0.0. Will be removed in version 5.0.0.", false)]
-    public class NLogFactory : NServiceBus.Logging.LoggingFactoryDefinition
```

Comment:
Isn't the upcoming version v4? (In that case shouldn't this be kept around to guide users?)

Reply:
@andreasohlund the type is still around but not the test exercising it because this was the only public API we exposed but the test can't use it anymore because it is obsoleted with error (you can't pragma disable that). But now TC fails because it doesn't discover tests so I have to look into that

Reply:
ah, so it was just the approval file that was deleted 👍

Reply:
I found a way to bring it back now without using the obsoleted API